web-components/src/components/organisms/ProjectFilters/ProjectFilters.tsx

```diff
@@ -0,0 +1,118 @@
+import { useState } from 'react';
+import { Trans } from '@lingui/macro';
+import {
+  Box,
+  ButtonBase,
+  Collapse,
+  Divider,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material';
+
+import { Subtitle, Title } from '../../typography';
+import TagFilter from './ProjectFilter.TagFilter';
+import CheckboxFilter from './ProjectFilters.CheckboxFilter';
+
+export interface FilterOptions {
+  name: string;
+  icon: JSX.Element;
+  id: string;
+}
+
+export interface Filter {
+  title: string;
+  displayType: 'tag' | 'checkbox';
+  options: FilterOptions[];
+  hasCollapse?: boolean;
+}
+
+export default function ProjectFilters({
+  filters,
+  activeFilterIds,
+  onFilterChange,
+  onFilterReset,
+}: {
+  filters: Filter[];
+  activeFilterIds: string[];
+  onFilterChange: (id: string) => void;
+  onFilterReset: () => void;
+}) {
+  const theme = useTheme();
+  const isXs = useMediaQuery(theme.breakpoints.down('sm'));
+  const [isExpanded, setIsExpanded] = useState<Record<number, boolean>>({});
+  const handleExpand = (index: number) => {
+    setIsExpanded({ ...isExpanded, [index]: !isExpanded[index] });
+  };
+  return (
+    <>
+      <Box display="flex" justifyContent="space-between" alignItems="baseline">
+        <Title variant="h4">
+          <Trans>Filters</Trans>
```

Comment:
`i18n` components and functions should not be used directly in `web-components` as the package should be agnostic to the `i18n` solution.